docset/winserver2012-ps/adfs/Get-ADFSRelyingPartyTrust.md

```diff
@@ -46,14 +46,14 @@ Gets the current property settings for a relying party trust that is specified b
 
 ### -------------------------- EXAMPLE 2 --------------------------
 ```
-C:\PS>Get-ADFSRelyingPartyTrust -Identifier https://SampleApp.SampleServer.org
+C:\PS>Get-ADFSRelyingPartyTrust -Identifier https://www.sampleserve.com/
 ```
 
 Description
 
 -----------
 
-Gets the current property settings for a relying party trust that is specified by the identifier https://SampleApp.SampleServer.org.
+Gets the current property settings for a relying party trust that is specified by the identifier https://www.sampleserve.com/.
```

Comment:
```suggestion
Gets the current property settings for a relying party trust that is specified by the identifier `https://www.sampleserve.com`.
```

Reply:
I'm sorry I missed this before, but example URLs should be styled as code so that they're not clickable. Inline would be surrounded by a single backtick.